packages/@glimmer/component/addon/index.ts

```diff
@@ -12,11 +12,11 @@ declare module '@ember/component' {
 import { setComponentManager } from '@ember/component';
 
 import GlimmerComponentManager from './-private/ember-component-manager';
-import _GlimmerComponent from './-private/component';
+import _GlimmerComponent, { Args } from './-private/component';
 import { setOwner } from '@ember/application';
 
-export default class GlimmerComponent extends _GlimmerComponent {
-  constructor(owner, args) {
+export default class GlimmerComponent<S> extends _GlimmerComponent<S> {
```

Comment:
Unless we provide a default for `S` here, this _requires_ users to specify a type arg, which is a breaking change. In fact, many tests no longer type-check with this change (try running `tsc` in the repo root).

Reply:
Ahhh, good catch. I thought I’d caught this everywhere I did it, but: no. Thank you!

Reply:
```suggestion
export default class GlimmerComponent<S = unknown> extends _GlimmerComponent<S> {
```

Reply:
I fixed it here, but/and I'm concerned that CI was passing without `tsc` checking! Will follow up on that front tomorrow.